src/read.jl

```diff
@@ -2,58 +2,45 @@
 function read_case(filepath)
     println("Reading from folder: " * filepath)
 
-    println("...loading case.mat")
-    case_mat_file = MAT.matopen(joinpath(filepath, "case.mat"))
-    mpc = read(case_mat_file, "mpc")
-
     # New case.mat analog
     case = Dict()
 
     # AC branches
-    # dropdims() will remove extraneous dimension
-    case["branchid"] = dropdims(mpc["branchid"]; dims=2)
-    # convert() will convert float array to int array
-    case["branch_from"] = convert(Array{Int,1}, mpc["branch"][:, 1])
-    case["branch_to"] = convert(Array{Int,1}, mpc["branch"][:, 2])
-    case["branch_reactance"] = mpc["branch"][:, 4]
-    case["branch_rating"] = mpc["branch"][:, 6]
+    branch = CSV.File(joinpath(filepath, "branch.csv"))
```

Comment:
Since we use `PyCall` anyways in `REISE.jl`, not sure whether we should look into reading pkl files directly in julia via `PyCall`.

Reply:
Interesting. It seems to be well supported.

Reply:
I think it's simpler to use csv as the common interface rather than something at the language level like PyCall. Not that it wouldn't necessarily work well, but I feel like relying only on a standard data format just rules out a potential class of issues that are inherent with that. We could always reevaluate if the explicit mapping gets to be too much maintenance..